app/views/layouts/_navigation_links.haml

```diff
@@ -1,5 +1,8 @@
-%li{:class => is_active_link?(page_path('about'))}= link_to "About", page_path('about')
-%li{:class => is_active_link?(page_path('contact'))}= link_to "Contact", page_path('contact')
+-# %li{:class => is_active_link?(page_path('about'))}= link_to "About", page_path('about')
+%li{:class => is_active_link?(page_path('welcome'))}= link_to "Welcome", '#welcome'
+%li{:class => is_active_link?(page_path('about'))}= link_to "About", '#about'
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace